### PR TITLE
[HOLD] Update Authentication and TestFramework reference to ADAL 2.28.3

### PR DESCRIPTION
**Pull request by cormacpayne**
Update Microsoft.Rest.ClientRuntime.Azure.Authentication to 2.2.9-preview
Update Microsoft.Rest.ClientRuntime.Azure.TestFramework to 1.4.1-preview

### CODE REVIEW

File: src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/project.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "version": "1.4.0-preview",
+  "version": "1.4.1-preview",
   "copyright": "Copyright (c) Microsoft Corporation",
```
Comment (hovsepm) [inline src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/project.json]: Assembly info for this file also needs to be updated.

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Comment by cormacpayne** [comment]
Cognitive Services: http://azuresdkci.cloudapp.net/view/2-.NET%20SDK/job/sdk-for-net-sign/1307/

KeyVault: http://azuresdkci.cloudapp.net/view/2-.NET%20SDK/job/sdk-for-net-sign/1308/

Search: http://azuresdkci.cloudapp.net/view/2-.NET%20SDK/job/sdk-for-net-sign/1309/

**Comment by cormacpayne** [comment]
Updated KeyVault: http://azuresdkci.cloudapp.net/view/2-.NET%20SDK/job/sdk-for-net-sign/1311/

**Comment by markcowl** [comment]
closing in favor of new PR

**Pull request closed without merging**